ibl_to_nwb/AlyxToNWB/alyx_to_nwb_converter.py

```diff
@@ -393,7 +365,7 @@ def create_acquisition(self):
                                     name=i['name'] + '_' + self.nwb_metadata['Probes'][j]['name'],
                                     starting_time=i['timestamps'][j][0, 1],
                                     rate=i['data'][j].fs,
-                                    data=H5DataIO(DataChunkIterator(iter_datasetvieww(i['data'][j])),
```

Comment:
try using the `buffer_size` arg of [`DataChunkIteractor`](https://hdmf.readthedocs.io/en/stable/hdmf.data_utils.html?highlight=DataChunkIterator#hdmf.data_utils.DataChunkIterator)

Reply:
does that work independently of `compression_level`, `shuffle ` of https://hdmf.readthedocs.io/en/latest/hdmf.backends.hdf5.h5_utils.html#hdmf.backends.hdf5.h5_utils.H5DataIO

Reply:
yes, it should

Reply:
have added buffer size parameter as input https://github.com/catalystneuro/IBL-to-nwb/pull/9/commits/6139858994ac8f0ad7ace448d8dd718aa4c07c21